mappings/net/minecraft/block/tile/TileEntityLootTable.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS none/ars net/minecraft/block/tile/TileEntityLootTable
```

Comment:
This isn't a tile entity class, right? It should be named differently to not confuse anyone, additionally, since the class is already a "loot table", maybe the methods don't all need the `LootTable` suffix?